securedrop/source_templates/use-tor-browser.html

```diff
@@ -0,0 +1,7 @@
+{% extends "base.html" %}
+{% block body %}
+<h1>You Should Use Tor Browser</h1>
+<p>If you are not using Tor Browser, you <strong>may not be anonymous</strong>.</p>
+<p>If you want to submit information to SecureDrop, we <strong>strongly advise you</strong> to install Tor Browser and use it to access our site safely and anonymously.</p>
+<p>Copy and paste the following address into your browser and follow the instructions to download and install Tor Browser: <pre>https://www.torproject.org/download.html.en</pre></p>
```

Comment:
I think this might be a better link to direct folks to instead: https://www.torproject.org/projects/torbrowser.html.en. I think the design is cleaner, and it has more detailed installation instructions (with screenshots!).

It might be worth hopping into the tor-dev IRC channel and asking what the "official" recommendation is.